backend/database/migrations/versions/7fcfe7937fd1_initial_clean_schema.py

```
@@ -0,0 +1,808 @@
+from pgvector.sqlalchemy import Vector
+from pgvector.sqlalchemy import Vector
```

Ruff (0.13.3)

Redefinition of unused `Vector` from line 1

Remove definition: `Vector`

(F811)